spec/quality_es_spec.rb

```diff
@@ -0,0 +1,78 @@
+# frozen_string_literal: true
```

Comment:
I think we need the `# encoding: utf-8` comment for ruby < 2 support

Reply:
Gotcha. Does that mean ruby < 2 doesn't encode unicode properly without that magical comment?

Reply:
ruby < 2 defaulted to interpreting source as ASCII, ruby 2+ default to utf8. Ruby < 2 handled unicode just fine, it just didn't default to it